bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Ephemeris.scala

```diff
@@ -30,7 +30,7 @@ final class Ephemeris(val site: Site, val compressedData: Deflated[List[(Long, F
     if (_task != null) _task.cancel()
     _task = new TimerTask {
       def run(): Unit = {
-        logger.info("Discarding inflated ephemeris.")
+        logger.fine("Discarding inflated ephemeris.")
```

Comment:
Lowering the priority of these log messages is unrelated but very helpful for debugging.